src/binary-reader.cc

```diff
@@ -1712,7 +1713,17 @@ Result BinaryReader::ReadLinkingSection(Offset section_size) {
                 CHECK_RESULT(ReadStr(&name, "symbol name"));
               switch (sym_type) {
                 case SymbolType::Function:
-                  CALLBACK(OnFunctionSymbol, i, flags, name, index);
+                  if (options_.read_linking_names != LinkingNameStrategy::DONT_USE_LINKING_NAMES) {
+                    bool prepend_dollar = (options_.read_linking_names == LinkingNameStrategy::USE_LINKING_NAMES_WITH_DOLLAR);
+
```

Comment:
No need for all these empty lines between statements, see surrounding code.

Reply:
I was trying to make it easier to review, but can trim it if it's more desirable :+1:

Reply:
GitHub diffs are pretty easy to read even without extra blank lines.